src/Controller/UI/ResearchGroupController.php

```diff
@@ -1,47 +1,53 @@
 <?php
 
-namespace Pelagos\Bundle\AppBundle\Controller\UI;
+namespace App\Controller\UI;
 
-use Pelagos\Bundle\AppBundle\Security\EntityProperty;
-use Pelagos\Bundle\AppBundle\Form\ResearchGroupType;
-use Pelagos\Bundle\AppBundle\Form\PersonResearchGroupType;
+use App\Entity\FundingOrganization;
+use App\Entity\ResearchGroup;
+use App\Handler\EntityHandler;
+use App\Security\EntityProperty;
+use App\Form\ResearchGroupType;
+use App\Form\PersonResearchGroupType;
 
-use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Component\Routing\Annotation\Route;
 
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
 
 /**
  * The Research Group controller for the Pelagos UI App Bundle.
  */
-class ResearchGroupController extends UIController implements OptionalReadOnlyInterface
+class ResearchGroupController extends AbstractController
 {
     /**
      * The Research Group action.
      *
-     * @param string $id The id of the entity to retrieve.
+     * @param EntityHandler $entityHandler The Entity Handler.
+     * @param integer       $id            The id of the entity to retrieve.
      *
-     * @Route("/research-group/{id}")
+     * @throws NotFoundHttpException When the research group was not found.
      *
-     * @throws createNotFoundException If research group could not be found.
+     * @Route("/research-group/{id}", name="pelagos_app_ui_researchgroup_default")
      *
      * @return Response A Response instance.
      */
-    public function defaultAction($id = null)
+    public function defaultAction(EntityHandler $entityHandler, int $id = null)
     {
         // Checks authorization of users
         if (!$this->isGranted('ROLE_DATA_REPOSITORY_MANAGER')) {
-            return $this->render('PelagosAppBundle:template:AdminOnly.html.twig');
+            return $this->render('template/AdminOnly.html.twig');
         }
 
         $ui = array();
         $ui['PersonResearchGroups'] = array();
 
         if (isset($id)) {
-            $researchGroup = $this->entityHandler->get('Pelagos:ResearchGroup', $id);
+            $researchGroup = $entityHandler->get(ResearchGroup::class, $id);
 
-            if (!$researchGroup instanceof \Pelagos\Entity\ResearchGroup) {
-                throw $this->createNotFoundException('The Research Group was not found!');
+            if (!$researchGroup instanceof \App\Entity\ResearchGroup) {
```

Comment:
full path not needed

Reply:
This is how it was before, who known why. It should still work. It can be changed if desired.